src/pivot_comb.jl

```diff
@@ -17,7 +19,9 @@ References:
 function pivot_comb(A,
                B::AbstractMatrix{T};
                tol::Float64=1e-8,
-               max_iter=30*size(A,2)) where {T}
+               max_iter=30*size(A,2),
+               P_init=nothing,
```

Comment:
This line should be something like
`P_init::Matrix{Bool} = falses(size(A,2), size(B,2))`
(check my sizes)